discovery-infra/tests/base_test.py

```diff
@@ -341,7 +343,41 @@ def kube_api_client(self):
 
     @pytest.fixture()
     def kube_api_context(self, kube_api_client):
-        kube_api_context = KubeAPIContext(kube_api_client, clean_on_exit=global_variables.test_teardown)
+        kube_api_context = KubeAPIContext(kube_api_client)
```

Comment:
Seems like a bad rebase. please look into this module for changes that were made and  aren't realted to the OLM changes

Reply:
yeah, will make sure to keep only changes we need.